charts/gatewayd/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "gatewayd.fullname" . }}-test-connection"
+  labels:
+    {{- include "gatewayd.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "gatewayd.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Is this where the healthz endpoints should be used?

Reply:
No it's not related to healthz endpoint (Liveness and Readiness). It's only used with `helm test` command. This chart is developed base on [v0.8.3](https://github.com/gatewayd-io/gatewayd/releases/tag/v0.8.3). In next release I'll add health endpoints to the chart.